src/responder/common/cache_req/cache_req.c

```diff
@@ -176,6 +176,11 @@ cache_req_validate_domain_enumeration(struct cache_req *cr,
     if (domain->enumerate == false) {
         CACHE_REQ_DEBUG(SSSDBG_TRACE_FUNC, cr, "Domain %s does not support "
                         "enumeration, skipping...\n", domain->name);
+	if(cr->rctx->enumeration_warn_logged == false)
```

Comment:
Coding style violation - we use the opening bracket `{` on the same line as the if.

Reply:
I would also like to add a `DEBUG` message next to the sss log message.